apps/omg_watcher/test/support/exit_processor/test_helper.ex

```diff
@@ -159,4 +176,21 @@ defmodule OMG.Watcher.ExitProcessor.TestHelper do
 
     long
   end
+
+  @doc """
+  Generates a given length of bytes for the given integer.
+  """
+  @spec binary_from_integer(integer(), integer()) :: binary()
+  def binary_from_integer(integer, num_bytes) do
```

Comment:
IMHO there is no gain having such functions. We usually put some non-sensical values e.g. <<1>> for binary, or
<<1::160>> to show it's 20-bytes length. KISS-rule

Whenever particular value isn't important we should make it look obviously non-sensical. Otherwise we're suggesting the value has  some more context.

Reply:
Thanks for the feedback @pnowosie. Incorporated it in 7f8da07.